zk_toolbox/crates/config/src/wallets.rs

```diff
@@ -40,6 +48,10 @@ impl WalletsConfig {
             fee_account: Wallet::empty(),
             governor: Wallet::empty(),
             token_multiplier_setter: Some(Wallet::empty()),
+            governance: Some(Wallet::empty()),
```

Comment:
Either we should add more wallets to config, which doesn't sound great

or we can distribute tokens directly through the script:
`l1-contracts/deploy-scripts/DeployZKAndBridgeToL1.s.sol
`
However, the issue distributing through the script is that two addresses are not know for Gateway chain, so potentially the script might be called 2 times, first to bridge the ZK token and distribute some balances, and then to distribute remaining balances after regsiterZkChain is ran for gateway